flake.nix

```
@@ -0,0 +1,35 @@
+{
+  description = "Fuzzy finder for Makefile";
```

I'm sorry. I rewrote description of this repository a while ago 😧
So please change the description field as follows.

```suggestion
  description = "fzf-make is the command line tool that execute make target using fuzzy finder with preview window.";
```